pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.6.2</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	
+	<groupId>ai.aitia</groupId>	
+	<artifactId>ah-auth-ruler</artifactId>
+	<version>4.4.1.0</version>
+	<packaging>jar</packaging>
+	<name>Arrowhead Authorization Ruler</name>
+	<description>Updates authorization rules in an Arrowhead Local Cloud</description>
+	
+	<url>https://github.com/Aitia-IIOT/ah-auth-ruler-script</url>
+	<licenses>
+    	<license>
+	     	<name>Eclipse Public License - v 2.0</name>
+	     	<url>https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt</url>
+	   </license>
```

Comment:
@borditamas

Not sure about this license.  I don't think the Eclipse Foundation has any business with this program.

Reply:
Agree. Let's use MIT:

```
<license>
      <name>MIT License</name>
      <url>http://www.opensource.org/licenses/mit-license.php</url>
    </license>
```